engines/pyfunc-ensembler-service/tests/conftest.py

```diff
@@ -11,25 +11,13 @@ def initialize(self, artifacts: dict):
 
     def ensemble(
             self,
-            features: Dict,
+            input: Dict,
             predictions: Dict,
             treatment_config: Dict) -> Any:
-        route_name_to_id = TestEnsembler.get_route_name_to_id_mapping(predictions)
         if treatment_config['configuration']['name'] == "choose_the_control":
-            return predictions[route_name_to_id['control']]['data']['predictions']
+            return predictions['control']['data']['predictions']
         else:
-            return predictions[0]['data']['predictions']
-
-    @staticmethod
-    def get_route_name_to_id_mapping(predictions):
-        """
-        Helper function to look through the predictions returned from the various routes and to map their names to
-        their id numbers (the order in which they are found in the payload.
-        """
-        route_name_to_id = {}
-        for i, pred in enumerate(predictions):
-            route_name_to_id[pred['route']] = i
-        return route_name_to_id
```

Comment:
Removed this unnecessary helper method since the `list` of route responses -> `dict` of route responses transformation is now all done behind the scenes by the `PyFunc` interface.